File: CoffeeBack/src/main/java/br/ufsm/csi/coffee/model/Pedido.java
package br.ufsm.csi.coffee.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Pedido {
    private int codigo;
    private String dataPedido;
    private int quantidade;
    private int comanda;
    private Produto produto;
    private Status status;
}

File: CoffeeFront/src/app/principal/pedido.ts
import { Produto } from "./produto/produto";
import { Status } from "./status";

export class Pedido{
    codigo?: number;
    produto?: Produto;
    quantidade?: number;
    status?: Status;
    dataPedido?: string;
    comanda?: number;

    constructor(codigo?: number, produto?: Produto, quantidade?: number, status?: Status, data_criada?: string, comanda?: number){
        this.codigo = codigo;
        this.produto = produto;
        this.quantidade = quantidade;
        this.status = status;
        this.dataPedido = data_criada;
        this.comanda = comanda;
    }
}